runtime/ast/function_declaration.go

```diff
@@ -26,8 +26,23 @@ import (
 	"github.com/onflow/cadence/runtime/common"
 )
 
+type FunctionPurity int
+
+const (
+	UnspecifiedPurity FunctionPurity = iota
+	ViewFunction      FunctionPurity = 1
```

Comment:
Enums in the codebase use "prefix-naming". Also, given that iota is used, no need to provide anything for the remaining cases.

```suggestion
	FunctionPurityUnspecified FunctionPurity = iota
	FunctionPurityView
```